ddtrace/contrib/snowflake/patch.py

```diff
@@ -14,6 +14,12 @@
     "snowflake",
     dict(
         _default_service="snowflake",
+        # FIXME: prefix snowflake span names with service
+        # The snowflake integration was introduced following a different pattern
+        # than all other dbapi-compliant integrations. It sets span names to
+        # `sql.query` whereas other dbapi-compliant integrations are set to
+        # `<service>.query`.
```

Comment:
`s/service/integration/g` ?

Reply:
I'd argue this is more correct and that it shouldn't be fixed, but that is my personal viewpoint.